README.md

```diff
@@ -79,7 +76,7 @@ in your program simply via:
 >>> IBMQ.load_account()
 ```
 
-Those who do not want to save their credentials to disk should use instead:
+Those who do not want to save their credentials locally should use instead:
```

Comment:
I think the former is clearer in this situation; "locally" can also mean "in memory" and you'd have to get the intent from the rest of the context.

Reply:
Oh, I just saw above that you and Julien were talking about this.  I still think it's worth using "disk" rather than locally here, because the next line is really trying to distinguish persistent and volatile storage.

Reply:
I agree with the emphasis on "persistency". So changed in 38690e56f7b19b07d245fd7ccc859fd6766ef156